### PR TITLE
refactor: simplify thirst

### PR DESCRIPTION
**Pull request by Laetta**

### CODE REVIEW

File: src/hydration/thirst.ts [hunk 1]
```diff
@@ -1,18 +1,97 @@
...
 import { PotionEffect, PotionEffectType } from 'org.bukkit.potion';
+import { addHydration, getHydration } from './expbar';
 
-// How often players' hydration level is decreased (seconds)
-const THIRST_CYCLE_DURATION = 60;
+/**
+ * How often players' hydration level is decreased (seconds)
+ * If duration is 60s and hydration -0.01, the bar will last 100 mins
+ */
+const THIRST_CYCLE_DURATION = 3;
```
Comment (raikasdev) [inline src/hydration/thirst.ts:10]: This probably should be 30? 5 minutes is a pretty short time period.

File: src/hydration/thirst.ts [hunk 2]
```diff
@@ -1,18 +1,97 @@
...
+}
+
+function becomeDehydrated(player: Player, hydration: number) {
+  dehydratedPlayers.add(player);
+
+  const msg = getDehydrationMessage(hydration);
+  player.sendTitle('', msg, 10, 40, 10);
+}
+
+function getDehydrationMessage(hydration: number) {
```
Comment (raikasdev) [inline src/hydration/thirst.ts:51]: Shouldn't this be made with the translation system?

**Review by raikasdev — CHANGES_REQUESTED** [review]
On top of those two, there are merge conflicts.

File: src/hydration/thirst.ts (hunk 1 above)
Comment (Laetta) [reply src/hydration/thirst.ts:10]: Oh, this should have been unchanged. I will change it back to 60

Comment (Laetta) [reply src/hydration/thirst.ts:10]: Oh, this should have been unchanged, so I will return it back to 60

File: src/hydration/thirst.ts (hunk 2 above)
Comment (Laetta) [reply src/hydration/thirst.ts:51]: This was made before translation system, so translations can be added later

**Pull request merged**